src/components/backend-ai-console.ts

```diff
@@ -1047,7 +1047,7 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
                   </mwc-list-item>
                   <mwc-list-item class="horizontal layout start center" @click="${() => this.splash.show()}">
                       <mwc-icon style="color:#242424;padding-right:10px;">info</mwc-icon>
-                      ${_t("console.menu.About")}
+                      ${_t("console.menu.AboutBackend.AI")}
```

Comment:
In translation part, dot (.) is used to divide namespaces. How about removing dot from AboutBackend.AI? (to AboutBackendAI)

Reply:
I am sorry for making so many mistakes, this time I believe that I have fixed every dot (.) instance. Thanks @inureyes for pointing these out to me, so patiently.